Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM webdevops/php-apache:ubuntu-17.04
```

Comment:
There is no need for a docker container in this project. Just use hugo.

Reply:
It's just so I can test it locally.

Reply:
I understand that, but like I said you don't need docker (or apache) for that . Just run `hugo server` in the site dir and you will have a running server at localhost:1313. It even hot replaces so you can edit and immediately see the changes.

Reply:
Oh. I didn't know that. Thanks.

Reply:
No worries. We should probably put that in the readme somewhere.

Reply:
Btw ive started migrating the doc stuff to the main git repo. Shall I migrate this branch as well? You'll have to check it out fresh if I do but that way we can have the migration done quickly. Alternative is to wait until this is ready to be merged

Reply:
If you just leave it hanging here I can migrate it over when you are done with migrating everything else. And I can keep working on it this weekend.